lexers/t/todotxt.go

```diff
@@ -0,0 +1,19 @@
+package t
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// Todotxt lexer. Lexer for Todo.txt <http://todotxt.com/> todo list format.
+var Todotxt = internal.Register(MustNewLexer(
+	&Config{
+		Name:      "Todotxt",
+		Aliases:   []string{"todotxt"},
+		Filenames: []string{"todo.txt", "*.todotxt"},
```

Comment:
Missing comment here for `filenames`.
https://github.com/pygments/pygments/blob/a590ac5ea7c00a41e253834306bfa19e38349c0b/pygments%2Flexers%2Ftextfmts.py#L211

Reply:
Good catch :+1: